flexget/plugins/filter/crossmatch.py

```diff
@@ -94,10 +97,31 @@ def entry_intersects(self, e1, e2, fields=None):
             log.trace('v1: %r' % v1)
             log.trace('v2: %r' % v2)
 
-            if v1 == v2:
-                common_fields.append(field)
+            if mode == "flexible":
+                v1Re = v1.lower()
+                v1Re = unicodedata.normalize('NFKD', v1Re).encode('ASCII', 'ignore')
+                v1Re = re.sub("[^a-zA-Z0-9 ]", " ", v1Re)
+                v1Re = v1Re.replace(' ','.*')
```

Comment:
I don't think adding new modes is the correct solution, it's just going to make this plugin really confusing to configure. Why not use a more specific character class like `\W` so you only match on whitespace?

I'm also still concerned about your choice to strip Unicode characters; this mode will be detrimental for, say, Russian content.